common/src/main/java/com/redhat/cloud/notifications/models/IntegrationTemplate.java

```diff
@@ -103,6 +123,7 @@ public enum TemplateKind {
         DEFAULT, // A fallback if nothing more specific is defined
         APPLICATION, // specific for one (sending) application
         EVENT_TYPE, // specific for an event_type of an application
-        ORG  // defined for a single (customer) organisation
+        ORG,  // defined for a single (customer) organisation
+        ALL // all kind of template can be acceptable
```

Comment:
Why is `ALL` needed?

Reply:
It's to be able to request any kind of template at once, no matter if it's an Org, event_type, application or default template lever.